db/schema.rb

```diff
@@ -17,77 +16,76 @@
   enable_extension "plpgsql"
 
   create_table "assignments", force: :cascade do |t|
-    t.string   "name",           null: false
-    t.jsonb    "metadata"
-    t.string   "workflow_id",    null: false
-    t.string   "subject_set_id", null: false
-    t.integer  "classroom_id",   null: false
-    t.datetime "created_at",     null: false
-    t.datetime "updated_at",     null: false
+    t.string "name", null: false
+    t.jsonb "metadata"
+    t.string "workflow_id", null: false
+    t.string "subject_set_id", null: false
+    t.bigint "classroom_id", null: false
```

Comment:
Yes, that was the PR that did it linked in my last comment. I used db:setup, but it was with an empty db/no scheme.rb on my rails5 branch. 

I can do the above and check in the resulting (more accurate?) scheme file when I get in.